.teamcity/src/main/kotlin/configurations/CompileAllBuildCacheNG.kt

```diff
@@ -8,10 +10,17 @@ class CompileAllBuildCacheNG(model: CIBuildModel, stage: Stage, oldCacheWithNgRe
         id("${model.projectId}_CompileAllBuild_NGRemote")
         name = "Compile All With NG Remote"
         description = "Compiles all production/test source code using old build cache but NG remote"
+
+        vcs {
+            checkoutDir = "ng-remote-${DslContext.uuidPrefix}"
```

Comment:
🤔 It seems like this ends up to be `build-cache-ng-Release`. Shall we add something here which VCS root we used as well? Or should this be good for now.

Reply:
This is not unique per VCS root. `Release` is not enough to qualify the VCS root. It is enough to qualify the VCS root for Gradle, but not within all VCS roots. For example, if GE decides to try out the build cache ng, the directory could easily be called `build-cache-ng-Release` again. That is why I'd expect to add more context to the custom checkout dir.